plugins/woocommerce-blocks/assets/js/base/components/product-name/index.tsx

```diff
@@ -10,6 +10,8 @@ import type { AnchorHTMLAttributes, HTMLAttributes } from 'react';
  */
 import './style.scss';
 
+type DisabledTagNameType = 'span' | 'h3';
```

Comment:
When would the `DisabledTagName` be a span if we are changing to h3?

Reply:
@alexflorisca I noticed that `cart-line-item-row.tsx` also uses Product Name, and may use `span` if a conditional is met. Let me know your thoughts, alternatively we could move it over to a `h3` but wanted to err on the side of caution.